.storybook/themes/commerce.scss

```diff
@@ -680,6 +680,46 @@
 						),
 					),
 				),
+				right-rail: (
+					display: grid,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+				),
+				right-rail-full-width-1: (
+					display: none,
+				),
+				right-rail-main: (
+					display: grid,
+					gap: var(--global-spacing-6),
+					max-width: initial,
+					width: calc(100vw - var(--global-spacing-8)),
+					margin: auto,
+				),
+				right-rail-full-width-2: (
+					margin: 0 calc(-1 * var(--global-spacing-5)) 0 0,
```

Comment:
Why are we using a negative margin?

Reply:
On mobile, it is aligned to the right of the screen

Reply:
@matthewroach I have updated the code base to reflect our discussion in huddle on Monday 7/18